specs/MultiProfile.md

```diff
@@ -29,6 +29,21 @@ Users can use this interface to get the cookie manager, which is shared by all W
 with this profile. The cookie manager got from profile (CoreWebView2.Profile.CookieManager) is the
 same as that got from CoreWebView2 (CoreWebView2.CookieManager).
 
+Currently, we already have **CoreWebView2Settings** interface to manage password-autosave and
+general-autofill, but it will work not immediately but after the next navigation, and it can only
```

Comment:
This paragraph contradicts [the documentation for CoreWebView2Settings](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2settings?view=webview2-winrt-1.0.1587.40), which says that the IsGeneralAutofillEnabled and IsPasswordAutosaveEnabled properties take effect immediately, not on next navigation.

> Changes to [CoreWebView2Settings.IsGeneralAutofillEnabled](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2settings?view=webview2-winrt-1.0.1587.40#isgeneralautofillenabled) and [CoreWebView2Settings.IsPasswordAutosaveEnabled](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2settings?view=webview2-winrt-1.0.1587.40#ispasswordautosaveenabled) apply immediately, while other setting changes made after [CoreWebView2.NavigationStarting](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?view=webview2-winrt-1.0.1587.40#navigationstarting) event do not apply until the next top-level navigation.

Reply:
This is because the documentation was changed and has been published for a while. If refer to earlier documentation (i.e., [1.0.1072.54](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2settings?view=webview2-winrt-1.0.1072.54)), the description does not explicitly exclude IsGeneralAutofillEnabled and IsPasswordAutosaveEnabled properties.

> Defines properties that enable, disable, or modify WebView features. Setting changes made after [CoreWebView2.NavigationStarting](https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2?view=webview2-winrt-1.0.1072.54#navigationstarting) event do not apply until the next top-level navigation.